docs/en/getting-started/get-started-docker.asciidoc

```diff
@@ -1,75 +1,300 @@
+security settings from your {es} cluster, authenticates to {es} with the
+`kibana` service account, and writes the security configuration to `kibana.yml`.
+
+[discrete]
+==== Prerequisites
+
+. Install the appropriate https://docs.docker.com/get-docker/[Docker application]
+for your operating system.
+
+. Pull the {es} Docker image.
 +
+--
 ifeval::["{release-state}"=="unreleased"]
-NOTE: Version {version} of {es} has not been released,
-so the sample compose file is not yet available for this version.
-See the {stack-gs-current}/get-started-docker.html[current version] for the latest sample files.
+
+WARNING: Version {version} of {es} has not yet been released, so no
+Docker image is currently available for this version.
+
 endif::[]
+
 ifeval::["{release-state}"!="unreleased"]
-["source","yaml",subs="attributes"]
---------------------------------------------
-include::docker/docker-compose.yml[]
---------------------------------------------
+
+[source,sh,subs="attributes"]
+----
+docker pull docker.elastic.co/elasticsearch/elasticsearch:{version}
+----
+
 endif::[]
-// tag::docker-memory[]
-. Make sure Docker Engine is allotted at least 4GiB of memory.
-In Docker Desktop, you configure resource usage on the Advanced tab in Preference (macOS)
-or Settings (Windows).
-// end::docker-memory[]
+--
+
+. Pull the {kib} Docker image.
++
+--
+ifeval::["{release-state}"=="unreleased"]
+
+WARNING: Version {version} of {kib} has not yet been released, so no
+Docker image is currently available for this version.
+
+endif::[]
+
+ifeval::["{release-state}"!="unreleased"]
+
+[source,sh,subs="attributes"]
+----
+docker pull docker.elastic.co/kibana/kibana:{version}
+----
+
+endif::[]
+--
+
+[[docker-start-enroll]]
+[discrete]
+==== Start a single-node {es} cluster and enroll {kib}
+
+The following command starts a single-node {es} cluster for development or
+testing.
+
+. Create an `elastic` network for your containers.
++
+[source,sh]
+----
+docker network create elastic
+----
+
+. Start {es} in Docker. A password is generated for the `elastic` user and
+output to the terminal, plus enrollment tokens for enrolling {kib} and adding
+additional nodes to your cluster.
++
+--
+ifeval::["{release-state}"=="unreleased"]
+
+WARNING: Version {version} of {es} has not yet been released, so no
+Docker image is currently available for this version.
+
+endif::[]
+
+ifeval::["{release-state}"!="unreleased"]
+[source,sh,subs="attributes"]
+----
+docker run --name es-node01 --net elastic -p 9200:9200 -it docker.elastic.co/elasticsearch/elasticsearch:{version}
+----
+
+endif::[]
+--
++
+TIP: You might need to scroll back a bit in the terminal to view the password
+and enrollment token.
 
-. Run `docker-compose` to bring up the three-node {es} cluster and {kib}:
+. Copy the generated password and enrollment token and save them in a secure
+location. These values are shown only when you start {es} for the first time.
 +
-["source","sh",subs="attributes"]
+[NOTE]
```

Comment:
I think I'd pull this info about resetting the elastic password out of the procedure into a separate subsection.

Reply:
I added a new section for generating passwords and enrollment tokens and moved the relevant information there. I'll probably make this change in the Elasticsearch docs as well.

Preview link: https://stack-docs_1888.docs-preview.app.elstc.co/guide/en/elastic-stack-get-started/master/get-started-stack-docker.html#docker-generate